cupy_xarray/accessors.py

```diff
@@ -9,9 +9,9 @@
 try:
```

Comment:
if you're worried about typing, then we might need this:
```suggestion
if TYPE_CHECKING:
    DuckArrayTypes = tuple[type[Any], ...]

dask_array_type: DuckArrayTypes
try:
```